example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/MainActivity.kt

```diff
@@ -66,6 +71,20 @@ fun Navigator(
             CardScheme.MASTERCARD,
             CardScheme.MAESTRO,
             CardScheme.AMERICAN_EXPRESS
+        ),
+        prefillData = PrefillData(
```

Comment:
Nit
since we are re-filling this field twice with the same data in the same class maybe we should have the object BillingFormAddress saved somewhere so we can reuse it as much as we want